problem1.py

```diff
@@ -1 +1,77 @@
+"""
+***************************************************
+Filename: LiveHack #3 problem1.py
+Description: Determine which group a player is in
+Author: Lee. M
+Date: 03/03/2021
+***************************************************
+"""
+# define variables
+****** Tournament Tracker ******
+Group_1_wins = int(input("Enter a number"))
+for i in range (5,6):
+  if i == 5:
+    print ("W")
+    print ("W")
+    print ("W")
+    print ("L")
+    print ("W")
+    print ("W")
+    print ("Your team is in Group 1")
```

Comment:
the sequence of wins and losses should be inputted by the user and not predetermined by the program. You should be COUNTING wins.